packages/IMAPClient-UI.package/ICFolderDialog.class/instance/update.st

```diff
@@ -1,10 +1,12 @@
 action - emails
 update
 
+	[self unconnectedAccounts isEmpty] whileFalse: [
+		(self login: self unconnectedAccounts first) ifFalse: [^false]].
 	self numberOfFetchedFolders: 0. 
 	self numberOfFolders: 
 		(self emailAccounts collect: [:each | each rootFolder recursiveChildFolders]) flatten size +
 		self emailAccounts size.
 
 	'Updating ' displaySequentialProgress: [self updateProcess].
-	self numberOfFetchedFolders: 0
\ No newline at end of file
+	self numberOfFetchedFolders: 0.
```

Comment:
probably no point at the end (according to code standards)

Reply:
will be fixed